lib/utils.ts

```diff
@@ -12,3 +13,23 @@ export function decimalToString(value: Decimal | null, defaultString: string = '
   }
   return converted.toString()
 }
+
+// Define a numerical order on Prisma enum, ordinarily done by string comparison
+export function ord(nextAction: NextAction): number {
+  switch (nextAction) {
+    case 'ADMIN_SCORING_MISSING_TMUA':
+      return 0
+    case 'ADMIN_SCORING_WITH_TMUA':
+      return 1
+    case 'PENDING_TMUA':
+      return 2
+    case 'REVIEWER_SCORING':
+      return 3
+    case 'UG_TUTOR_REVIEW':
+      return 4
+    case 'INFORM_CANDIDATE':
+      return 5
+    case 'CANDIDATE_INFORMED':
+      return 6
+  }
+}
```

Comment:
Why not putting these values into a constant list and implementing `ord` as a one-liner `nextActions.findIndex(elem)`?